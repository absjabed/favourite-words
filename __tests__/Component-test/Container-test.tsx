import 'react-native';
import React from 'react';
import Container from '../../src/components/Container';
import renderer, { create } from 'react-test-renderer';

it('Container renders correctly', () => {
  renderer.create(<Container />);
});

test('Container snapshot', () => {
  const tree = create(<Container/>);
  expect(tree).toMatchSnapshot();
});
