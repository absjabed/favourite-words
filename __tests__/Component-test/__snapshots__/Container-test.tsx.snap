// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "padding": 20,
        "paddingHorizontal": 30,
      },
      undefined,
    ]
  }
/>
`;
